parquet-variant-compute/Cargo.toml

```
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "parquet-variant-compute"
+# This package is still in development and thus the version does
+# not follow the versions of the rest of the crates in this repo.
+version = "0.1.0"
+license = { workspace = true }
+description = "Apache Parquet Variant Batch Processing"
+homepage = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
+keywords = ["arrow", "parquet", "variant"]
+readme = "README.md"
+edition = { workspace = true }
+# needs a newer version than workspace due to
+# rror: `Option::<T>::unwrap` is not yet stable as a const fn
+rust-version = "1.83"
+
+
+[dependencies]
+arrow = { workspace = true }
+arrow-schema = { workspace = true }
+parquet-variant = { path = "../parquet-variant" }
```

Can you please make this "workspace=true" (otherwise cargo publish gets angsty)?

```suggestion
parquet-variant = { workspace = true }
```

---

```
error: failed to parse manifest at `/Users/harsh.motwani/arrow-rs/parquet-variant-compute/Cargo.toml`

Caused by:
  error inheriting `parquet-variant` from workspace root manifest's `workspace.dependencies.parquet-variant`

Caused by:
  `dependency.parquet-variant` was not found in `workspace.dependencies`
```

---

Adding `parquet-variant = { path = "./parquet-variant" }` to the root Cargo.toml gets it working, so that's what I will do for now.